devops/Makefile

```diff
@@ -43,6 +43,110 @@ define DB_NAME
 "SIMSDB"
 endef
 
+define FORMIO_FORMS_NAME
+"forms-flow-forms"
+endef
+
+define FORMIO_BPM_NAME
+"forms-flow-bpm"
+endef
+
+define FORMIO_API_NAME
+"forms-flow-api"
+endef
+
+define FORMIO_WEB_NAME
+"forms-flow-web"
+endef
+
+define HAMONGO_DB_TEMPLATE_PATH
+"openshift/forms-flow-ai/database/mongo-ha.yml"
+endef
+
+define HAMONGO_DB_PARAM_PATH
+"openshift/forms-flow-ai/database/mongo-ha-param.yml"
+endef
+
+define SECRETS_PATRONI_DB_TEMPLATE_PATH
+"openshift/forms-flow-ai/secrets/ffa-patroni-secrets.yml"
+endef
+
+define CREATE_FFA_DB_TEMPLATE_PATH
+"openshift/forms-flow-ai/database/create-ffa-db.yml"
+endef
+
+define FFA_BPM_DB_NAME
+"FFA_BPM_DB"
+endef
+
+define FFA_API_DB_NAME
+"FFA_API_DB"
+endef
+
+define SECRETS_TEMPLATE_PATH
+"openshift/forms-flow-ai/secrets/secrets.yml"
+endef
+
+define SECRETS_PARAM_PATH
+"openshift/forms-flow-ai/secrets/secrets-param.yml"
+endef
+
+define FORM_AI_BUILD_TEMPLATE_PATH
+"openshift/forms-flow-ai/build.yml"
+endef
+
+define FORM_AI_FORMS_DEPLOY_TEMPLATE_PATH
+"openshift/forms-flow-ai/forms-deploy.yml"
+endef
+
+define FORM_AI_FORMS_URL
+"$(HOST_PREFIX)aest-ffa-forms.$(DOMAIN)"
+endef
+
+define FORM_AI_BPM_DEPLOY_TEMPLATE_PATH
```

Comment:
There are some variables like that declared and used only once. Is this something that will change or potentially could be used in other places along with the same file?

Reply:
its used only once, but its declared at the top for better readability of where the BPM, FORMS, WEB, API deploy paths are.